packages/patternfly-4/react-core/src/components/Dropdown/DropdownItem.js

```diff
@@ -1,169 +1,44 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
 import PropTypes from 'prop-types';
-import dropdownStyles from '@patternfly/react-styles/css/components/Dropdown/dropdown';
-import appLauncherStyles from '@patternfly/react-styles/css/components/AppLauncher/app-launcher';
-import { css } from '@patternfly/react-styles';
+import InternalDropdownItem from './InternalDropdownItem';
```

Comment:
This is confusing, but the contents in this file got replaced with the previous Item.js contents (which was then deleted), and the previous Item.js contents got moved into InternalDropdownItem.js